### ISSUE TITLE
Server not allowing Slack authentication

### ISSUE THREAD

**Issue by pll** [leighmcculloch/looks.wtf#13]
Sorry if this isn't the place to log this issue, but the website doesn't list any contact info.

I've been trying to add looks.wtf to my slack channel since yesterday. All I get is:

```
    Error: Server Error
    The server encountered an error and could not complete your request.
    Please try again in 30 seconds.
```
Thanks.

**Comment by leighmcculloch** [comment]
Oh no! 😨

**Comment by geilt** [comment]
Same, was so stoked to see the Slack Integration, then the 500 I was like nooooooo.

**Comment by leighmcculloch** [comment]
Sorry all, for some reason App Engine seems to be erroring before it even gets to booting the app that does oauth and the slack commands. It's erroring in the runtime startup and complaining about syntax errors in scripts that have nothing to do with my app. 😢

**Comment by leighmcculloch** [comment]
Okay, I think I've figured it out. Looks like I broke the thing, and the way it broke I guess it causes this strange errors during app boot.

**Comment by leighmcculloch** [comment]
Fixed in 3bd6504. Thanks for letting me know @pll and @geilt!

**Comment by pll** [comment]
@leighmcculloch Awesome! I can confirm it's working!  Thanks a bunch for fixing :)
